### PR TITLE
fix(ivy): ViewRef.detachFromAppRef should clean the DOM

### PR DESCRIPTION
**Pull request by marclaval**
This fixes FW-1127.

### CODE REVIEW

File: packages/core/src/render3/node_manipulation.ts [hunk 1]
```diff
@@ -243,6 +243,15 @@ export function addRemoveViewFromContainer(
   }
 }
 
+/**
+ * Detach a `LView` from the DOM by detaching the its nodes.
```
Comment (kara) [inline packages/core/src/render3/node_manipulation.ts]:
```suggestion
 * Detach a `LView` from the DOM by detaching its nodes.
```

**Review by kara — APPROVED** [review]
LGTM, sans typo

File: packages/core/src/render3/node_manipulation.ts (hunk 1 above)
Comment (marclaval) [reply packages/core/src/render3/node_manipulation.ts]: Fixed

**Comment by kara** [comment]
[presubmit](http://test/OCL:237245727:BASE:237245754:1551973464007:e45a1d93)

**Pull request closed without merging**